demos/python_demos/human_pose_estimation_demo/models.lst

```diff
@@ -0,0 +1,4 @@
+# This file can be used with the --list option of the model downloader.
+human-pose-estimation-0002
+human-pose-estimation-0003
+human-pose-estimation-0004
```

Comment:
human-pose-estimation-0001 is not listed there, Is that intentionally? If not, and all human-pose-estimation models are supported, please use human-pose-estimation-???? instead of full list of particular models

Reply:
Correct. human-pose-estimation-0001 is completely different and not supported by this demo.

Reply:
we think it is bad choice to have two different models (although solving the same task) with almost the same naming and as a result two different demo application, with the same naming again, just implemented in C++ and python. I would recommend to adopt existing demo to use new models as well as old one. Otherwise it's going to confuse customers. We will not accept such PR.
@IRDonch @fzhar @eizamaliev @snosov1 FYI

Reply:
Good to know. Though, looking at the current list of demos, I'd assume that renaming the demo should work as well, right?

Facts:
* C++ mask_rcnn_demo and Python instance_segmentation_demo do not support the common set of models and showcase Mask R-CNN like models.
* There are tones of demos for different object detection "models (although solving the same task) with almost the same naming".

Reply:
fortunately we do not have `tones` of object detection demos, only 5 for python and 2 for C++, and we are going to unify them (see [PR-1639](https://github.com/openvinotoolkit/open_model_zoo/pull/1639) and [PR-1744](https://github.com/openvinotoolkit/open_model_zoo/pull/1744)). If you can support old human-pose-estimation-0001 model in new python demo that would be helpful.

Reply:
@vladimir-dudnik Added human-pose-estimation-0001 to this new Python demo. See the notes in the first comments for more details on that matter.